### ISSUE TITLE
Support non-FQCNs in phpdocs

### ISSUE THREAD

**Issue by brendt** [spatie/value-object#13]
Right now, you need to write the whole classname in the `@var` type declaration:

```php
/** @var \Spatie\ValueObject\T */
```

Imported namespaces should also be supported.

**Comment by brendt** [comment]
There's no good way to do this, unfortunately. Maybe someone comes up with a good idea, but for now we'll close it.

**Comment by mfn** [comment]
My suggestion then is to not close this issue:
- people are asking for it
- it's not solved
- (just because we don't have a good solution yet, doesn't mean there won't be a fix)

Just my 2c

**Comment by brendt** [comment]
You're free to submit a PR with a solution though, we'll keep that open and can discuss specifics there. I just don't think of this as "an issue": we've thought about this before, and came to the conclusion that we don't want to add support for it in the library, unless someone comes up with a better idea. That better idea belongs in a PR 🙂

Sidenote: when we discussed this a week ago, this package was still only recently released. I realise that since than, more people have started looking at it and I understand it should be an active discussion again. So let's continue this in the original PR, which I've reopened: https://github.com/spatie/data-transfer-object/pull/14
